### ISSUE TITLE
vg msga global?

### ISSUE THREAD

**Issue by hgibling** [vgteam/vg#647]
![image](https://cloud.githubusercontent.com/assets/7570351/22528986/9e504da4-e8a3-11e6-9023-4e52af33501b.png)

Is there a relatively simple way to either:
- build the graph allowing for a SNP at position 4? Force a global alignment?
- merge nodes 1 and 71 at TGT but keep separate nodes for G and A?

Or would I have to manually edit the graph in JSON?

This is from an msga build of 22 equal-length sequences that are quite similar, so I'm hoping to avoid having to manually edit all of the paths.

**Comment by jeizenga** [comment]
@adamnovak This looks like the artifact that the full length alignment bonus is supposed to handle. Is it exposed in the msga interface for non-threaded alignments?

**Comment by adamnovak** [comment]
Yeah, the right solution might be to add an end bonus in MSGA. It's complicated, though, by the fact that an end bonus really only ought to apply to the outermost bands in a banded alignment (right?). Or should we apply it to all of them?

**Comment by jeizenga** [comment]
I don't see much harm in letting apply to all of them. I doubt it's really _necessary_ in the inner bands because the overlap between bands will generally resolve the soft clips that happen without it. But I also can't think of a reason it would harm anything.

**Comment by hgibling** [comment]
Ack, sorry! False alarm. I was using an older version of vg, but it works fine with `v1.4.0-2142-gfe5e050`
![image](https://cloud.githubusercontent.com/assets/7570351/22607770/5d9fca52-ea27-11e6-8d0e-4ff469901600.png)

**Comment by faithokamoto** [comment]
Seems this was handled.
